### PR TITLE
Second attempt at styling

### PR DESCRIPTION
**Pull request by jonathanrstern**

### CODE REVIEW

**Comment by jonathanrstern** [comment]
https://www.loom.com/share/2f0aacc6e146428abd5ca11d09c0622d

File: app/assets/stylesheets/BreedTile.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.breed-tile-name a {
+  text-decoration: none;
+  color: #333333;
+}
+
+.breed-tile-name a:visited {
+  // CHANGE COLOR AFTER LUNCH
+  color: #333333;
```
Comment (dovakeen118) [inline app/assets/stylesheets/BreedTile.scss:23]: Is it after lunch? 😂

File: app/assets/stylesheets/NavBar.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.menu-link a {
+  padding: 0px;
+  text-decoration: none;
+}
+
+.menu-link a:visited {
+  color: #000080;
+}
```
Comment (dovakeen118) [inline app/assets/stylesheets/NavBar.scss:60]: This is not a required change, but a really neat thing you can do specifically with SCSS files is nest styles just like how the elements themselves are nested!
```
.menu-link {
  // styles
  
  a {
    // styles
  }
  
  a:visited {
    // styles
  }
}
```

Of course this does couple elements together, like the above example for an `a` tag styles only apply when it's found within a `menu-link`. A similar approach that you've taken here, but in more of an SCSS manner

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- *= require foundation_and_overrides
-
- */
+* This is a manifest file that'll be compiled into application.css, which will include all the files
```
Comment (dovakeen118) [inline app/assets/stylesheets/application.scss:2]: Let's make sure this still has the same formatting, with the body indented

File: app/assets/stylesheets/BreedShow.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  font-size: 1em;
+  margin: 0px;
+}
+
+.breed-rating {
+  font-style: italic;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-size: 1em;
+}
```
Comment (dovakeen118) [inline app/assets/stylesheets/BreedShow.scss:21]: A consideration - There's a lot of styles here that are repeated for elements that are siblings - we can create a class for the div that wraps around this data to DRY up some of the repeated styles

File: app/javascript/react/components/PetIndexTile.js [hunk 5]
```diff
@@ -3,10 +3,15 @@ import { Link } from "react-router-dom"
 
 const PetIndexTile = (props) => {
   return (
-    <div>
-        <Link to={`/pets/${props.id}`}>
-            {props.name}
+    <div className="pet-tile">
+      <Link className="pet-tile-box" to={`/pets/${props.id}`}>
+        {/* {props.photo_url} */}
```
Comment (dovakeen118) [inline app/javascript/react/components/PetIndexTile.js:8]: We can remove unused code

File: app/javascript/react/components/PetShowContainer.js [hunk 6]
```diff
@@ -39,9 +39,12 @@ const PetShowContainer = (props) => {
 
     return(
         <div>
-          <h1>{pet.name}</h1>
-          <h3>{pet.description}</h3>
-          {breedTiles}
+          {/* add photo of pet */}
```
Comment (dovakeen118) [inline app/javascript/react/components/PetShowContainer.js:42]: Let's remove code comments about future features, reminders like this live best on the story for the feature it's for

**Review by dovakeen118 — COMMENTED** [review]
A few comments to remove/ unused code to remove and an indentation fix to the manifest file!

**Pull request merged**